docs/ops/deployment/kubernetes.md

```diff
@@ -166,18 +242,31 @@ kind: Service
 metadata:
   name: flink-jobmanager
 spec:
+  type: ClusterIP
   ports:
   - name: rpc
     port: 6123
   - name: blob
     port: 6124
-  - name: query
-    port: 6125
   - name: ui
     port: 8081
   selector:
     app: flink
     component: jobmanager
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: flink-jobmanager-svc
+spec:
+  type: NodePort
```

Comment:
@Myasuka, do you really need to expose the service as a `NodePort`?
Shouldn't the above `ClusterIP` type `flink-jobmanager` service be enough for cluster to work properly?

Reply:
Yes, `ClusterIP` should be enough for the cluster to work properly. But without a `NodePort` on jobmanager's rest RPC port, I couldn't find a way to submit my application via `bin/flink run -m <k8s-flink-cluster>`. It seems I could only submit my application via web UI which is not so convenient for developers.

Reply:
If your situation is in some test/development setup, you can try to:
 * run `kubectl port-forward ${flink-jobmanager-pod} 8081:8081` to make JM reachable from your local machine (where you run `bin/flink run -m ...`).
 * I don't know well the `bin/flink run -m ${endpoint}` functionality, but if it doesn't work with urls that have additional path, the script/program behind it could have been extended to support such endpoints (smt. like: `bin/flink -m "http://localhost:8081/api/v1/namespaces/default/services/flink-jobmanager:ui/proxy" ...`).

If the Kubernetes resources and the documentation here are served as some recommended/exemplar Flink cluster setup, then adding a service definition that exposes node port would be a bad example, imo.
I suggest to remove the new service definition and maybe just add some more options (in the documentation) how developers can make JM reachable outside of test K8s cluster.

Reply:
Okay, I think we might add a separate service yaml file in the documentation to represent the `rest service` and tell users different ways to submit jobs: via `kubectl port-forward` or via the nodeport of this rest service.